content/physical_properties/electrical_conductivity/electrical_conductivity_lab_setup_measurements.rst

```diff
@@ -1,225 +1,209 @@
+
+Electrical resistivity measurements can be performed in both the frequency domain and the time domain. For frequency domain measurements, a sinuisoidal current is driven through the rock sample. In the time domain, a boxcar waveform is used.
+
+.. figure:: ./images/cond_lab_setup.png
+   :figwidth: 75%
    :align: center
-   :name: GSCcompleximpedance
+   :name: fig_cond_lab_setup
+
+   Basic setup for electrical resistivity measurements. (A) Theoretical circuit where rock acts as a a resistive element. (B) Rock sample between two electrodes. (C) Circuit schematic for frequency domain and time domain measurements.
 
-   Complex impedance measurement system at Geological Survey of Canada (GSC).
+DC Conductivity/Resistivity
+---------------------------
 
-.. figure:: ./images/ACcircuit_simple.png
-   :figwidth: 30%
-   :align: left
-   :name: ACcircuit_simple
+.. figure:: ./images/DCcircuit_simple.png
+   :figwidth: 25%
+   :align: right
+   :name: fig_DC_measurements
 
-   A simple AC circuit.
+   Electrical circuit for DC resistivity measurements.
 
-An equivalent circuit diagram for the lab measurements is shown in
-:numref:`ACcircuit_simple`. For a source that is providing a voltage, the
-currents depend upon the conductivity of the rock. The electrical impedance is
-defined as
+The DC (or zero-frequency) resistivity is obtained by forcing direct current (:math:`I`) through the rock and measuring the voltage drop (:math:`V`). Using Ohm's law, the resistance (:math:`R`) due to the rock is given by:
 
 .. math::
+   R = \frac{V}{I}
 
-   Z(\omega) = \frac{V(\omega)}{I(\omega)}
+According to Pouillet's law, the electrical resistivity of the sample is the product of the resistance and its cross-sectional area (:math:`A`), divided by the sample's length (:math:`l`):
+
+.. math::
+   \rho = \frac{RA}{l}
+
+The DC resistivity is a single, real-valued quantity and has units Ohm meters (:math:`\Omega m`).
+
+Frequency-Domain Measurements
+-----------------------------
+
+Electrical resistivity can be frequency-dependent and complex-valued, which
+effectively generates polarization effects with the applied electric field.
+This is often called induced polarization (IP) effects. In order to measure
+complex conductivity of a rock specimen, we inject sinusoidal currents (that is, AC
+currents) into to the rock at logarithmically spaced frequencies and measure the corresponding voltages. The typical frequency range for these measurement is from 0.016 Hz - 1 MHz.
```

Comment:
this is a pretty specific frequency range (eg 0.016Hz). Do you have a citation for this?